components/stepper/stories/template.js

```diff
@@ -50,6 +67,7 @@ export const Template = ({
 		>
 			${Textfield({
 				...globals,
+				size,
 				placeholder: "1",
```

Comment:
Should we remove placeholder from this example? We don't have one in the docs and it makes it look like the field value is 1. Maybe set a default value of 1 or 0 in the field instead?

I also noticed that the input being rendered in Storybook isn't `type="number"` (allows letters to be inputted).

Reply:
Good catches there! I removed placeholder and am passing the number type now to textfield.